backend/src/config/auth.rs

```diff
@@ -42,6 +43,7 @@ impl Config {
     /// * If the auth server is unreachable or is set up incorrectly.
     #[allow(clippy::expect_used)]
     pub async fn init(app_config: &crate::config::app::Config) {
+        debug!("Fetching endpoints from discovery endpoint...");
```

Comment:
I think it is rather useless to log that something, that cannot fail anyway, will happen.

Reply:
I thought it might be interesting to put this there to see how the app behaves a what it does when. But I'll change it to trace then.

Reply:
Why do we need it all? Maybe if you log the number of endpoints it could be somewhat useful.

Reply:
Well that's what debug and trace are for right? To understand the flow of the application and what it does. For example when the setup doesn't work it might be interesting to see what it executed before that.

Reply:
> * which logging system is used in doc/architecture/08concepts.md (in separate PR or in [Documentation/architecture #385](https://github.com/ElektraInitiative/PermaplanT/pull/385))

@markus2330 Should I still do something here or is the current doc enough?